### PR TITLE
No recipes

### PR DESCRIPTION
**Pull request by mariakleiner**
update Arcs API and Graph app and CommonApp to directly run graphs (skipping the intermediate recipes represenation)

### CODE REVIEW

File: core/ts/recipe/ParticleCook.ts [hunk 1]
```diff
@@ -49,4 +49,7 @@ export class ParticleCook {
   static async derealizeParticle(runtime: Runtime, arc, node: ParticleNode) {
     arc.removeHost(node.id);
   }
+  static async removeParticles(arc, particleIds: string[]) {
```
Comment (sjmiles) [inline core/ts/recipe/ParticleCook.ts:52]: nice

File: core/ts/recipe/StoreCook.ts [hunk 2]
```diff
@@ -94,6 +94,12 @@ export class StoreCook {
     runtime.removeStore(meta.name);
     arc.removeStore(meta.name);
   }
+  static async removeStores(runtime: Runtime, arc: Arc, storeNames: string[]) {
+    storeNames.forEach(name => {
```
Comment (sjmiles) [inline core/ts/recipe/StoreCook.ts:98]: nice

File: pkg/Apps/Graph/Library/NodeGraphApp.js [hunk 3]
```diff
@@ -5,7 +5,7 @@
  * license that can be found in the LICENSE file.
 */
```
Comment (sjmiles) [inline pkg/Apps/Graph/Library/NodeGraphApp.js:6]: GraphMaker is the beginning of converting Graph! to CommonApp+Graph format. So, the old one is deprecated and the new one isn't ready yet. :)

File: pkg/Apps/Database/Library/DbApp.js [hunk 4]
```diff
@@ -6,10 +6,8 @@
  * license that can be found in the LICENSE file or at
  * https://developers.google.com/open-source/licenses/bsd
  */
```
Comment (sjmiles) [inline pkg/Apps/Database/Library/DbApp.js:8]: Another goal is to convert DbApp to CommonApp+Graph format. If we have a habit of using GraphMaker objects in App Graphs, there is a virtuous cycle in favor of NodeTypes.

File: pkg/Apps/Graph/Library/NodeGraphApp.js (hunk 3 above)
Comment (mariakleiner) [reply pkg/Apps/Graph/Library/NodeGraphApp.js:6]: I'll start on this tomorrow, now that i have a better grasp of the CommonApp+Graph combo!

**Comment by mariakleiner** [comment]
i think it's ready to be looked at and submitted (modulo the conflicts, which i'll resolve shortly)

**Review by sjmiles — APPROVED** [review]
Nice work, thank you

**Pull request merged**